clients/include/testing_gemm.hpp

```diff
@@ -89,18 +89,8 @@ void testing_gemm(const Arguments& arg)
     rocblas_int ldb = arg.ldb;
     rocblas_int ldc = arg.ldc;
 
-    T h_alpha;
-    T h_beta;
-    if(std::is_same<T, rocblas_half>{})
-    {
-        h_alpha = float_to_half(arg.alpha);
-        h_beta  = float_to_half(rocblas_isnan(arg.beta) ? 0 : arg.beta);
-    }
-    else
-    {
-        h_alpha = arg.alpha;
-        h_beta  = rocblas_isnan(arg.beta) ? 0 : arg.beta;
-    }
+    T h_alpha = arg.get_alpha<T>();
+    T h_beta  = rocblas_isnan(arg.get_beta<T>()) ? 0.0 : arg.get_beta<T>();
```

Comment:
`get_beta<T>()` should already check for NaN and return 0.0 if beta==NaN.

Reply:
understand. This is my mistake. will update it on third commit "update complex change"